vscode/webviews/CodyPanel.tsx

```diff
@@ -106,7 +106,9 @@ export const CodyPanel: FunctionComponent<
                         />
                     )}
                     {view === View.Prompts && <PromptsTab setView={setView} />}
-                    {view === View.Account && <AccountTab setView={setView} />}
+                    {view === View.Account && !clientCapabilities.isVSCode && (
```

Comment:
```suggestion
                    {view === View.Chat && clientCapabilities.isVSCode && (
```

If you change the code to this, and then change the `view === View.Chat` to `view === View.Account` above, you can then debug it in VS Code  :D